### PR TITLE
Sebastian Podgorski - Spell checker for class names

### PR DESCRIPTION
**Pull request by wenaro**

### CODE REVIEW

File: violetplugin-classdiagram/src/main/java/com/horstmann/violet/product/diagram/classes/edge/SpellChecker.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//package com.horstmann.violet.product.diagram.classes.edge;
+//
+///**
+// * Created by Wenaro on 2017-01-07.
+// */
+//public class SpellChecker {
+//
+//    public static void check(){
+//        JLanguageTool langTool = new JLanguageTool(new BritishEnglish());
```
Comment (orchowski) [inline violetplugin-classdiagram/src/main/java/com/horstmann/violet/product/diagram/classes/edge/SpellChecker.java]: YAGNI

File: violet-framework/src/main/java/com/horstmann/violet/product/diagram/abstracts/node/INodeName.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.horstmann.violet.product.diagram.abstracts.node;
+
+import com.horstmann.violet.product.diagram.abstracts.IColorable;
+import com.horstmann.violet.product.diagram.property.text.LineText;
+
+/**
+ * @author Sebastian Podgorski
+ */
+public interface INodeName extends INode, IColorable {
```
Comment (orchowski) [inline violet-framework/src/main/java/com/horstmann/violet/product/diagram/abstracts/node/INodeName.java:9]: Already exists and is called INamedNode.

**Review by orchowski — CHANGES_REQUESTED** [review]
You have to resolve conflicts, and use already existing interface(remove your's)

File: violetplugin-classdiagram/src/main/java/com/horstmann/violet/product/diagram/classes/edge/SpellChecker.java (hunk 1 above)
Comment (wenaro) [reply violetplugin-classdiagram/src/main/java/com/horstmann/violet/product/diagram/classes/edge/SpellChecker.java]: I dont know exactly why its here but i will remove it

**Review by abizukojc — CHANGES_REQUESTED** [review]
Build on Travis failed. Fix it.

**Pull request closed without merging**